OrbitGl/AsyncTrack.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "AsyncTrack.h"
+
+#include "App.h"
+#include "FunctionUtils.h"
+#include "GlCanvas.h"
+#include "ManualInstrumentationManager.h"
+#include "TimeGraph.h"
+#include "capture_data.pb.h"
+
+using orbit_client_protos::FunctionInfo;
+using orbit_client_protos::TimerInfo;
+
+AsyncTrack::AsyncTrack(TimeGraph* time_graph, const std::string& name) : TimerTrack(time_graph) {
+  SetName(name);
+  SetLabel(name);
+}
+
+[[nodiscard]] std::string AsyncTrack::GetBoxTooltip(PickingId id) const {
+  const TextBox* text_box = time_graph_->GetBatcher().GetTextBox(id);
+  if (text_box == nullptr) return "";
+  ManualInstrumentationManager* manager = time_graph_->GetManualInstrumentationManager();
+  orbit_api::Event event = manager->ApiEventFromTimerInfo(text_box->GetTimerInfo());
+
+  const FunctionInfo* func =
```

Comment:
Is it documented somewhere that these are "fake" functions? Even if it is, I would suggest to add a reminder here. Reading this without that knowledge, the code here looks wrong ("why is it getting function names in the case of manual instrumentation?"). 

Something like:
"Reminder that for manually instrumented scopes, we record the name as a fake function name with a high address outside of the range of possible regular function addresses"

WDYT?

Reply:
Oh, actually, this is not exactly what is done here. I think this does need a comment though. Essentially, the FunctionInfo here will be one of the empty stubs in Orbit.h. It is used here to add information about the module from which the timer comes from.

Reply:
Actually, I added a temporary variable, module_name, to make it clearer.

Reply:
Ah, I see, makes sense!

Reply:
I added a comment.